include/cuda/std/detail/libcxx/include/__config

```diff
@@ -648,8 +648,8 @@ typedef __char32_t char32_t;
 #define _LIBCUDACXX_HAS_VIRTUAL_DESTRUCTOR(...) __has_virtual_destructor(__VA_ARGS__)
 #endif
 
-#if __check_builtin(is_literal)
-#define _LIBCUDACXX_IS_LITERAL(...) __is_literal(__VA_ARGS__)
+#if __check_builtin(is_literal_type)
```

Comment:
Regarding `__check_builtin` -> `__has_builtin` changes that used to be here. The `__has_builtin` has different result compared to `!__is_identifier` for some compilers. Here's a [reference](https://godbolt.org/z/hvcK1hEja).